examples/uplink_set.go

```diff
@@ -36,15 +37,16 @@ func main() {
 
 	// Create Uplink Set
 	networkUris := new([]utils.Nstring)
-	*networkUris = append(*networkUris, utils.NewNstring("/rest/ethernet-networks/58196a2d-0043-4281-9728-7c973d9660b9"))
+	ethernet_ntw, _ := ovc.GetEthernetNetworkByName(ethernet_network)
+	*networkUris = append(*networkUris, ethernet_ntw.URI)
 
 	fcNetworkUris := make([]utils.Nstring, 0)
 	fcoeNetworkUris := make([]utils.Nstring, 0)
 	portConfigInfos := make([]ov.PortConfigInfos, 0)
 	privateVlanDomains := make([]ov.PrivateVlanDomains, 0)
 
 	uplinkSet := ov.UplinkSet{Name: new_uplink,
-		LogicalInterconnectURI:         utils.NewNstring("/rest/logical-interconnects/5fe6ef52-42d6-4895-8500-dfb633478225"),
+		LogicalInterconnectURI:         utils.NewNstring("/rest/logical-interconnects/f2587e79-c4de-4c71-8e0d-f8178258dfa6"),
```

Comment:
Can we use LI name and use get call to get the LI URI?

Reply:
done